FundacionParaguaya/MentorApp
# Map ui update

This is an improvement to the map ui to show a map even when no location is available

**To Test**
1. While logged in, turn all internet access off. 
2. Go to Family Location screen.
3. After a brief time you should see a `Map not available` message with no map or accuracy showing. Draft coordinates remain `null`.
4. Turn internet access on go back to Family Location screen.
5. If the GPS returns no location after 10 secs you should still see a map (centered by default near Africa, zoomed out). To simulate no location returned either get sim card out of your device or better yet, go to this line https://github.com/FundacionParaguaya/MentorApp/blob/map-ui-update/src/screens/lifemap/Location.js#L113 and set a very low timeout like `1` for example. You should be able to drag the map around and press the centering button to try and find your location within 10 seconds again.
6. Otherwise the map should function as normal.

**Related issues**
- Resolves #193
